tests/fixtures/saas/jira_fixtures.py

```diff
@@ -0,0 +1,148 @@
+    }
+
+
+@pytest.fixture(scope="session")
+def jira_identity_email(saas_config):
+    return (
+        pydash.get(saas_config, "jira.identity_email") or secrets["identity_email"]
+    )
+
+
+@pytest.fixture(scope="session")
+def jira_identity_phone_number(saas_config):
+    return (
+        pydash.get(saas_config, "jira.identity_phone_number")
+        or secrets["identity_phone_number"]
+    )
+
+
+@pytest.fixture(scope="function")
+def jira_erasure_identity_email() -> str:
+    return f"{cryptographic_util.generate_secure_random_string(13)}@email.com"
+
+
+@pytest.fixture
+def jira_config() -> Dict[str, Any]:
+    return load_config_with_replacement(
+        "data/saas/config/jira_config.yml",
+        "<instance_fides_key>",
+        "jira_instance",
+    )
+
+
+@pytest.fixture
+def jira_dataset() -> Dict[str, Any]:
+    return load_dataset_with_replacement(
+        "data/saas/dataset/jira_dataset.yml",
+        "<instance_fides_key>",
+        "jira_instance",
+    )[0]
+
+
+@pytest.fixture(scope="function")
+def jira_connection_config(
+    db: Session, jira_config, jira_secrets
+) -> Generator:
+    fides_key = jira_config["fides_key"]
+    connection_config = ConnectionConfig.create(
+        db=db,
+        data={
+            "key": fides_key,
+            "name": fides_key,
+            "connection_type": ConnectionType.saas,
+            "access": AccessLevel.write,
+            "secrets": jira_secrets,
+            "saas_config": jira_config,
+        },
+    )
+    yield connection_config
+    connection_config.delete(db)
+
+
+@pytest.fixture
+def jira_dataset_config(
+    db: Session,
+    jira_connection_config: ConnectionConfig,
+    jira_dataset: Dict[str, Any],
+) -> Generator:
+    fides_key = jira_dataset["fides_key"]
+    jira_connection_config.name = fides_key
+    jira_connection_config.key = fides_key
+    jira_connection_config.save(db=db)
+
+    ctl_dataset = CtlDataset.create_from_dataset_dict(db, jira_dataset)
+
+    dataset = DatasetConfig.create(
+        db=db,
+        data={
+            "connection_config_id": jira_connection_config.id,
+            "fides_key": fides_key,
+            "ctl_dataset_id": ctl_dataset.id,
+        },
+    )
+    yield dataset
+    dataset.delete(db=db)
+    ctl_dataset.delete(db=db)
+
+
+@pytest.fixture(scope="function")
+def jira_create_erasure_data(
+    jira_connection_config: ConnectionConfig, jira_erasure_identity_email: str
+) -> None:
+
+    jira_secrets = jira_connection_config.secrets    
+    base_url = f"https://{jira_secrets['domain']}"
+
+    # user
+    body = {
+            "name": "Ethyca Test Erasure",
+            "emailAddress": jira_erasure_identity_email,
+    }
+    
+    users_response = requests.post(
+        url=f"{base_url}/rest/api/3/user", 
+        json=body,
+        auth=HTTPBasicAuth(
+            f"{jira_secrets['username']}", f"{jira_secrets['api_key']}"
+        ),
+        )
+    user = users_response.json()
+
+    sleep(30)
```

Comment:
bit of a nit, but could we use the `saas_test_utils.poll_for_existence` pattern here? just helps to reduce any unnecessary waiting time will still providing the guarantee that the data you've created exists.

see https://github.com/ethyca/fides/blob/main/tests/fixtures/saas/braze_fixtures.py#L192 for an example (even though that example _also_ has a `sleep`, the goal is really to avoid any hardcoded `sleep`s if possible...